middleware/secure.go

```diff
@@ -53,6 +53,13 @@ type (
 		// trusted web page context.
 		// Optional. Default value "".
 		ContentSecurityPolicy string `yaml:"content_security_policy"`
+
+		// CSPReportOnly would use the `Content-Security-Policy-Report-Only` header instead
+		// of the `Content-Security-Policy` header. This allows iterative update of the
+		// content security policy by only reports the violations that would
```

Comment:
Shouldn't it be reporting?

Reply:
As in `CSPReporting` ?

Reply:
@vishr I think `CSPReporting` might cause slight confusion with the `report-uri` part in the CSP header, so I kept it as this. I'm open to changing it if you feel so. Thoughts?

Reply:
or... were you referring to the typo in the comment for `CSPReportOnly`? I just realized there was a typo. After correcting it and while pushing the change, it dawned on me that maybe I had completely misunderstood your initial comment 😂